index.bs

```diff
@@ -12089,6 +12114,102 @@ running the algorithm for an {{AudioNode}}, using an <a>input
 buffer</a> and the value(s) of the {{AudioParam}}(s) of this
 {{AudioNode}} as the input for this algorithm.
 
+<h3 id="interruption-handling">Handling an interruption on the {{AudioContext}}</h3>
+
+An <dfn>interruption</dfn> is an event generated by the user agent when it needs
+to halt audio playback for an {{AudioContext}}. For example, The user agent may
+create an interruption when another application requests exclusive access to the
+audio output hardware.
+
+When an [=interruption=] happens, the user agent must <a>queue a control message</a>
+to interrupt the {{AudioContext}}.
+
+Running a <a>control message</a> to <dfn attribute for="AudioContext"
+lt="interruption-start">interrupt an {{AudioContext}}</dfn> |context|
+means running these steps on the <a>rendering thread</a>:
+
+1. If the |context|'s {{[[rendering thread state]]}} is <code>closed</code>
+    or <code>interrupted</code>:
+
+    1. Abort these steps.
```

Comment:
No sublist when doing this, just inline the substep.

Reply:
Done!